x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_table.tsx

```diff
@@ -27,17 +28,24 @@ export const EngineTable: ReactFC<IEngineTableProps> = ({
   data,
   pagination: { totalEngines, pageIndex = 0, onPaginate },
 }) => {
-  const { enterpriseSearchUrl } = useContext(KibanaContext) as IKibanaContext;
+  const { enterpriseSearchUrl, http } = useContext(KibanaContext) as IKibanaContext;
+  const engineLinkProps = {
+    href: `${enterpriseSearchUrl}/as/engines/${name}`,
```

Comment:
Did you intend to drop the `color="primary"` attribute? It looks like we lost it on the "Manage" link.

Reply:
Oh hmm, I wonder why I had that? I think that was from an older mock when I was using the "actions" column (I ended up dropping it because the actions column on mobile is meant for icons, not text). Setting color primary doesn't appear to do anything now, so I think this is good to drop.

Thanks for the eagle eye Jason!